spec/adapters/ci_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "spec_helper"
+
+module Suspenders
+  module Adapters
+    RSpec.describe Ci do
```

Comment:
Mmm, you can ignore those as it's a bit silly. I think by now it's fixed in the original.